app/controllers/api/v1/ecf_users_controller.rb

```diff
@@ -51,7 +51,7 @@ def email
       end
 
       def users
-        users = User.is_ecf_participant
+        users = User.where(id: ParticipantProfile::ECF.joins(:teacher_profile).select("teacher_profiles.user_id"))
```

Comment:
Is suspect typing this everywhere is going to get boring. Not one for this PR, but would we be able to refactor this to some kind of helper or service object? C# would have ParticipantService that goes and gets the participants

Reply:
Actually, the logic here is a bit odd. We are inside of `users_controller` - why do we filter users to ecf only? It would make sense for participants_controller, but inside user_controller it is a sign of violated boundaries...

Reply:
Eeeh, we are in `ecf_users_controller`, so filtering them to be just ecf users if alright imo. That's a recent change, prior to that it was simply named poorly :)

Reply:
Ah, must have changed on merge with development :D So another note - there are no ECF users, there are ECF participants and that's what we should be serializing here. But that's for later